pages/news/2020-05-16-intro-outro-sections.html

```diff
@@ -0,0 +1,141 @@
+In this case, I watch the scrolling waveform when the outro start point is coming up on the old track.
+Then, I press play on the new track when the old track reaches the outro start.
+
+<a href="/static/images/news/intro-outro/starts-aligned-waveforms.png">
+<img src="/static/images/news/intro-outro/starts-aligned-waveforms.png" style="max-width:100%;" alt="Two tracks' waveforms with the starts of the outro and intro aligned"/>
+</a>
+
+When I load a new track, I compare the times of the intro & outro to decide whether I want to line up the start points or the end points.
+There are no firm rules I use to decide which method to use; it's an artistic judgement based on the musical content of each track, what the vibe of the crowd is at that moment, and what I want to do with the mix.
+
+If I do not have the intro end and outro start points marked when I load a track, before I mix in the track, I find and mark these while listening in my headphones.
+Once I have these marked, I seek back to the intro start to get the track ready.
+If I don't have time to find the outro end point before mixing in the track, I can use the new deck cloning feature to drag and drop to a new deck, then seek ahead on the new deck and listen in my headphones to mark the outro start point.
+If I don't have time to do that either, I right click on the overview waveform where I think the outro starts and look how far that is from the end of the track to compare it to the length of the intro of the next track.
+
+<a href="/static/images/news/intro-outro/outro-right-click.png">
+<img src="/static/images/news/intro-outro/outro-right-click.png" style="max-height: 137px; max-width: 100%;" alt="Right clicking on the overview waveform to approximate the length of the outro"/>
+</a>
+
+Here I see that the outro is approximately 30 seconds long. However, I also see that the analyzer placed the outro end point before the end of the track, so the outro is actually a little bit shorter than 30 seconds. Without marking the outro start point, I would have to do some math in my head to find exactly how long the outro is.
+
+## Intro & Outro Cues With AutoDJ ##
+
+In Mixxx 2.3, these new intro & outro section cues can be used to tell AutoDJ how long to crossfade and how to align the tracks in time.
+AutoDJ uses the intro & outro cues in the new "Full Intro + Outro" and "Fade At Outro Start" modes.
+In both modes, instead of crossfading over an arbitrary number of seconds, AutoDJ compares the duration of the outro of the old track and the intro of the new track.
+AutoDJ picks the time of the shorter section as the crossfade time.
+
+If the outro is shorter than the intro, AutoDJ will align the start of the outro with the start of the intro in both modes.
+
+<a href="/static/images/news/intro-outro/autodj-align-starts.png">
+<img src="/static/images/news/intro-outro/autodj-align-starts.png" style="max-width:100%;" alt="AutoDJ aligning the start of the intro and outro"/>
+</a>
```

Comment:
Here and below: please include the AutoDJ controls row in the screenshot so that the images are easier to relate to the transition modes.